### PR TITLE
Add support for the PYTHONFAULTHANDLER environment variable.

### LINKED ISSUE #14: Support PYTHONFAULTHANDLER env var
Would you be interested in having support for that? If yes, I can make a PR for that (some `.pth` trickery a la https://pypi.python.org/pypi/hunter).

### PR DESCRIPTION
**Pull request by ionelmc**
Fixes #14.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -9,6 +9,9 @@ memory on the stack, even on stack overflow (not available on Windows).
 
 Import the module and call faulthandler.enable() to enable the fault handler.
 
+Alternatively you can set the PYTHONFAULTHANDLER environment variable to a
+non-empty value.
```
Comment (haypo) [inline README]: You should also modify doc/

File: faulthandler.pth [hunk 2]
```diff
@@ -0,0 +1 @@
+import os, sys; exec("if os.environ.get('PYTHONFAULTHANDLER'):\n import faulthandler\n faulthandler.enable()")
```
Comment (haypo) [inline faulthandler.pth]: Why do you use exec? And you may add "try/except ImportError" as in hunters, no?

File: tests.py [hunk 3]
```diff
@@ -337,6 +337,15 @@ def test_enable_single_thread(self):
             'Segmentation fault',
             all_threads=False)
 
+    def test_enable_env_var(self):
+        self.check_fatal_error("""
+            import faulthandler
+            faulthandler._sigsegv()
```
Comment (haypo) [inline tests.py]: I would prefer a simple test checking if the module is enabled. See test_faulthandler.py in CPython:
https://hg.python.org/cpython/file/e4802f19eee4/Lib/test/test_faulthandler.py#l309

File: tox.ini [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32
+envlist = py{26,27,32}
```
Comment (haypo) [inline tox.ini]: This change must do done in a separated commit and you must require the minimum tox version supporting {...} syntax. Sorry, I don't remember the minimum version.

File: faulthandler.pth (hunk 2 above)
Comment (ionelmc) [inline faulthandler.pth]: Because this is not valid syntax:
```
import foo; if blabla: do(stuff)
```
The `if` is disallowed in multistatement lines.

**Comment by ionelmc** [comment]
All fixed now.

**Comment by haypo** [comment]
Thanks, I merged your patch.

**Pull request merged**